### PR TITLE
Views

### PR DESCRIPTION
**Pull request by IngDavidTM**
# Blog app - Views

This is the sixth milestone of the 'Blog app' project. It has four tables, `users`, `comments`, `likes` and `posts`, each of them has some methods and was linked using foreign ids. Also, it has some tests and validations for each model and controller as well as pages that redirect to `Users` and `Posts`.

It uses:

- `Linters`
- `Ruby on rails`

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find_by(id: params[:user_id])
+    @posts = Post.where(author_id: params[:user_id])
+  end
```
Comment (Davidosky007) [inline app/controllers/posts_controller.rb:5]: * Based on the wireframe provided your post-controller index page is supposed to render 5 most recent comments as shown in the wireframe below 👇 **HINT** 💭  you can create a method that will fetch the 5 most recent comments
image

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,5 +1,12 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+    @posts = Post.where(author_id: params[:id])
+    @comments = Comment.where(posts_id: params[:id])
+  end
```
Comment (Davidosky007) [inline app/controllers/users_controller.rb]: * Based on the wireframe provided your user-controller index page is supposed to render 3 most recent posts as shown in the wireframe below 👇👇 **HINT** 💭  you can create a method that will fetch the 3 most recent posts
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @IngDavidTM 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
* Kindly work on the requested changes 🙏 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @IngDavidTM , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**